src/WarcraftLegacies.Source/Quests/Warsong/QuestOrgrimmar.cs

```diff
@@ -32,12 +31,11 @@ public sealed class QuestOrgrimmar : QuestData
     public override string RewardFlavour => "The city of Orgrimmar was finally constructed by the Warsong engineers, it is now a home for the new Horde and a symbol of power and innovation. The Warchief has rewarded us generously for our work!";
 
     /// <inheritdoc/>
-    protected override string RewardDescription => $"Control of all units in Orgrimmar, {GoldReward} gold, able to train Varok and Azerite Siege Engines";
+    protected override string RewardDescription => $"Control of all units in Orgrimmar, gold, able to train Varok and Azerite Siege Engines";
```

Comment:
Looks like a typo. It shouldn't mention gold at all right?

Reply:
oh yeah